src/main/java/com/google/devtools/build/lib/starlarkbuildapi/ActionApi.java

```diff
@@ -134,6 +134,15 @@ public interface ActionApi extends StarlarkValue {
               + " settings which are only pre-set in the execution environment.")
   Dict<String, String> getEnv(StarlarkSemantics semantics) throws EvalException;
 
+  @StarlarkMethod(
+      name = "env_inherit",
```

Comment:
nit: include a more explicit description of the return type. I think it's a list of strings (env var names), right? But the phrasing almost sounds like it might be a map the environment variables

Reply:
Done